### DISCUSSION TITLE
Where can I access the unminified developer version of Play CDN?

### DISCUSSION THREAD

**Discussion by hepsinioku** [tailwindlabs/tailwindcss#10386] [Help]
Where can I find the unminified developer version of Play CDN?
 
link
https://cdn.tailwindcss.com/

Thanks.

**Comment by adamwathan** [comment]
Hey! The project isn’t open source so the source isn’t available.

**Reply by drschwabe** [reply]
any plans to open source this ?

**Comment by hepsinioku** [comment]
It would be really nice if there was an open source developer version.

webpack, vite vs. I want to get the css output of the code I wrote without using it. CDN does this successfully. But it renders as inline style.
I was looking for a small edit to output the css code as an array instead of accessing the inline style code.

Why do I need something like this;
While the site is active, every time I add a new component to a project I'm developing, I have to regenerate the css file. But if I can get the css output of the component instantly, I can only include the necessary css codes without compiling.

I hope something like this is supported.

**Comment by humandad** [comment]
If the Play CDN project is closed source, how is the code licensed? Is the minified code MIT?

**Reply by adamwathan** [reply]
Hey! The code is just proprietary with no license, just like the minified JS files served by any other site you visit.

**Reply by humandad** [reply]
got it. Thanks!

**Comment by IdaParks** [comment]
you may be able to access the unminified developer version of Play CDN by checking the documentation or support resources provided by the CDN service. Alternatively, you could try reaching out to their customer support team for assistance in locating the unminified version. They should be able to provide guidance on how to access it, or if it is not available publicly, they may be able to provide access to it for your account[.
](https://www.mybkexperience.onl/)
